packages/leavitt-elements/src/leavitt-person-company-select.ts

```diff
@@ -126,14 +134,23 @@ export class LeavittPersonCompanySelectElement extends LoadWhile(LitElement) {
     this.searchTerm = '';
   }
 
+  /**
+   *  Sets focus on the input.
+   */
   focus() {
     this.textfield.focus();
   }
 
+  /**
+   *  Returns true if the input passes validity checks.
+   */
   checkValidity() {
     return this.textfield.checkValidity();
   }
 
+  /**
+   *  Runs checkValidity() method, and if it returns false, then it reports to the user that the input is invalid.
+   */
```

Comment:
LoadWhile is showing up as a public method.
Looks like adding `/**
 * @ignore
 */`

above line 331 where it is called, hides it from the docs

IsLoading also shows up in the docs, but not sure where to add the ignore to get rid of it

Reply:
Nevermind, even the first one doesn't appear to be removed by adding @ignore